### ISSUE TITLE
CreateAsync method of UserResource returns 422

### ISSUE THREAD

**Issue by harsheet** [justeat/ZendeskApiClient#89]
CreateAsync method of UserResource always return 422 even when the user is created on Zendesk. Can someone help with this?

Thanks

**Comment by EmmanuelAtx** [comment]
have you tried with our pull request? For us, all requests needed to be inside a wrapper class.
https://github.com/justeat/ZendeskApiClient/pull/87
{"identity":{"id": ... }}
When examined with fiddler, we found out that the first part was missing:
{"identity":{

**Comment by harsheet** [comment]
I checked the branch and I think its only fixed for Identity, but I was using CreateAsync of UserResource that needs to be inside a User wrapper. Like this -

{"user":{"name": "Roger12 Wilco", "email": "roge12@yahoo.com"}}

**Comment by EmmanuelAtx** [comment]
Indeed, we haven't time to fix the whole project. So we just did what we need. We already found the bug.  We hope someone will have the time to fix this in the whole project, so many people can enjoy the fix :-)

**Comment by harsheet** [comment]
yes indeed :)

**Comment by mikerogers123** [comment]
Hi @harsheet, is this still an issue for you? I am conscious this issue is 5 years old and there have been many versions introduced since then
